### ISSUE TITLE
Completely redesign everything

### ISSUE THREAD

**Issue by dellsystem** [dellsystem/wikinotes#133]
- Better, less generic design
- Clean up stylesheets/js (incorporate elements and ideas from the Twitter bootstrap, but don't rely on it too much)

This will be the big frontend overhaul this summer. Should be done in time for an official launch before the fall semester starts. **EDIT**: nope. for winter 2013?

Related issues: #7, #11, #49, #81, #110 

I should upgrade to Bootstrap v3 while I'm at it.

**Comment by hc5** [comment]
just filing under here, better table of contents

**Comment by hc5** [comment]
hmm i'm bored so I'm gonna write down a few design problems I see that are easy to tackle, i'm no designer so feel free to call me out on my shit:
- The topbar contains links that are mostly useless, the only two being helpful are Courses/Help, Home can be removed entirely because there is a bigger link for the homepage directly under it. News has its own row for the latest news(or wtf is even the point of news). About/contributing can be put in the footer maybe. When there are only Courses/Help left, we can perhaps merge the header and the topbar, saving a lot of vertical space.
- The rightbar class looks pretty shit. Lists inside the topbar are not distinctly seperated, looking at this ![](http://i.imgur.com/V0mRF.jpg), for items that have titles that are multiple lines, it's not clear whether it is part of that item, or another item entirely. Also, teal on light grey is pretty hard to read.
- The courses pages is also pretty bad. All the links that say 'click here' or are part of a sentence are terrible and should be eliminated entirely, instead, the headings are enough as a link. In fact, I'm not even sure why there needs to be even text under each heading, they are descriptive enough. Also, is the 10 random courses really useful/necessary?

that's about it, sorry if i sound nitpicky

**Comment by dellsystem** [comment]
agreed about the right bar, updating that has been on my to-do list for a while. never thought about the topbar that way but i guess you're right, it can definitely be restructured. the courses page will also be completely redesigned.

**Comment by zafarali** [comment]
Hi I'm trying to improve my CSS skills. How can I help?

**Comment by dellsystem** [comment]
Hey Zafarali! This is actually a pretty big task that I was planning to take on myself. The best way you could help is by looking through the website and finding aspects of it that could be improved, design-wise. IMO, the best way to improve your CSS skills is to build something and design it yourself, so if there's something you want to build I would suggest doing that - it's a great way to learn. Thanks for asking!

**Comment by zafarali** [comment]
Thanks, I'm right now working with certain elements and I wanted to redo the edit/preview box on wikinotes.ca I've been using the site semi-frequently these past few days and the box just doesn't make me want to write more!

I know its superfluous and abstract but I think it's a really important part of a site whose main aim is to encourage pople to share the notes online!

**Comment by dellsystem** [comment]
For sure, the editing box is pretty terrible at the moment. There is an
outstanding issue on incorporating the ACE editor (#170) which @wetmore was
assigned to but it doesn't look like he's made any progress on it. If you
want to take that feel free
